### ISSUE TITLE
Private videos are not visible in local channels

### ISSUE THREAD

**Issue by Vertux** [Chocobozzz/PeerTube#5539]
### Describe the current behavior

if you make a video private and assign it to a channel, these private videos do not show up in a channel. 
In the channel overview, they were counted for the channel, but when you open the channel itself, the counter shows zero videos and none private video is listed - see screenshots:

### **Channellist:**
<img width="338" alt="Bildschirm­foto 2023-01-15 um 15 46 52" src="https://user-images.githubusercontent.com/10457918/212550553-c7960bc5-3a37-45db-8d5e-7aa7ed0aec3d.png">

### **Channel:**
<img width="391" alt="Bildschirm­foto 2023-01-15 um 15 48 22" src="https://user-images.githubusercontent.com/10457918/212550596-4771a875-5347-49be-824d-223391d32b3a.png">

### **Filtered videos (channel):**
<img width="555" alt="media1" src="https://user-images.githubusercontent.com/10457918/212550667-b4e0297c-ad2b-4491-a626-7b8033b18edf.png">

### Steps to reproduce

1. Create a channel
2. Import a video via URL e.g. YouTube and mark it as private, wait until it is finished (or mark any public video as private and assign it to the channel)
3. Go to My Library-->Channels. The channel information shows one video.
4. Now click the channel name. The channels does not list any video and says "no video".
5. Go to  My Library-->Videos and filter for the channel name. The imported video is shows up and is assigned to the right channel, is playable, an everything else is fine too.

### Describe the expected behavior

Private videos should be counted and listed in their associated channel.

### Additional information

* PeerTube instance:
  * Version: 5.0.1

* Firefox 108.0.2, macOS

**Comment by DalekDave** [comment]
Yes, I'd really like to be able to see and view my Private videos in a user-friendly way in my profile's pages, and not just in the Videos page as at present. I support this feature request.

**Comment by agru** [comment]
In addition to “Private” videos, those with the “Scheduled” or “Password protected” setting are also affected. For the video- and channel-management of power users with a lot of videos, for example accounts of a bigger organization, this is a bigger problem.

**Comment by Chocobozzz** [comment]
@agru Can you explain why the "My videos" with channels filters doesn't work for your use case?

> So I am sorry, but for us this is not just a feature request

If it's important for your organization and want to help to fund this feature, you can contact us by email (in my github profile).

**Comment by agru** [comment]
Channel filter in my videos is indeed a workaround for the problem. Anyway ... I will contact you about funding.

**Comment by Chocobozzz** [comment]
FYI we're also redesigning the "My videos" page so it's easier to list and manage videos with quick channel filters. (should be available in the following weeks/months)

**Comment by agru** [comment]
That sounds great. Anyway, I just sent you a mail ...
